models/solana_utils/solana_utils_daily_balances.sql

```diff
@@ -40,7 +41,7 @@ WITH
                   WHERE tx_success 
                   AND block_time > now() - interval '1' day
                   {% if is_incremental() %}
-                  AND block_time >= date_trunc('day', now() - interval '1' day)
+                  AND {{incremental_predicate('block_time')}}
```

Comment:
we shouldn't use the macro here as well, sorry for the confusion

Reply:
oh okay got it